web/app/routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require("express");
+var router = express.Router();
+
+// frontend routes =========================================================
+// route to handle all angular requests
+router.get("*", function(req, res) {
+  res.sendfile("./public/index.html");
+});
+
+module.exports = router;
```

Comment:
I have not touched the "server.js" during my work on the endpoints. Will this change have to be handled elsewhere?